CHANGELOG

```diff
@@ -0,0 +1,13 @@
+[ChangeLog]
+
+A file to keep a log of changes made in this project.
```

Comment:
User [semantic versioning](https://semver.org/) or something similar for proper logs.